clients/java/signalr/src/main/java/com/microsoft/signalr/WebSocketTransport.java

```diff
@@ -82,8 +83,8 @@ public void setOnClose(Consumer<String> onCloseCallback) {
     }
 
     @Override
-    public CompletableFuture<Void> stop() {
-        return webSocketClient.stop().whenComplete((i, j) -> logger.info("WebSocket connection stopped."));
+    public Completable stop() {
+        return webSocketClient.stop().doOnEvent(t -> logger.info("WebSocket connection stopped."));
```

Comment:
Why is this `doOnEvent` vs `doOnComplete`?

Reply:
We still want to log even if there was a websocket error on close

Reply:
Ah, so `onEvent` fires in either case. That's the part I was missing :).